GDJS/Runtime/capturemanager.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-2016 Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  export type CaptureOptions = {
+    screenshots?: {
+      timing: number;
+      signedUrl: string;
+      publicUrl: string;
+    }[];
+  };
+
+  /**
+   * Manage the captures (screenshots, videos, etc...) that need to be taken during the game.
+   */
+  export class CaptureManager {
+    _gameRenderer: gdjs.RuntimeGameRenderer;
+    _captureOptions: CaptureOptions;
+
+    constructor(
+      renderer: gdjs.RuntimeGameRenderer,
+      captureOptions: CaptureOptions
+    ) {
+      this._gameRenderer = renderer;
+      this._captureOptions = captureOptions;
+    }
+
+    /**
+     * To be called when the scene has started rendering.
+     */
+    setupCaptureOptions(isPreview: boolean): void {
+      if (!isPreview || !this._captureOptions.screenshots) {
+        return;
+      }
+
+      for (const sscreenshotCaptureOption of this._captureOptions.screenshots) {
+        setTimeout(async () => {
+          console.info(
+            `Taking a screenshot and uploading it to ${sscreenshotCaptureOption.publicUrl}`
+          );
+          await this.takeScreenshot(sscreenshotCaptureOption.signedUrl);
+        }, sscreenshotCaptureOption.timing);
+      }
+    }
+
+    /**
+     * Helper function to convert a base64 string to a Blob, which can be uploaded to a server.
+     */
+    base64ToBlob(base64) {
```

Comment:
uber nitpicking: if it's a helper, you might as well declare it outside of the class so we don't start seeing games using gdjs.CaptureManager.base64ToBlob and complaining when we remove/rename this later. The less API surface, the better.